src/components/Copy/index.tsx

```diff
@@ -14,6 +14,9 @@ const ClipboardContainer = styled(Box)`
 	font-size: 0.875em;
 	line-height: ${(props) =>
 		px(props.theme.lineHeight * props.theme.fontSizes[2])};
+	&:hover {
+		visibility: visible;
+	}
```

Comment:
I checked out the respective UI branch and couldn't see any difference w/ or w/o this line.
Can you expand why we needed this?

Reply:
I was trying to replicate the properties of the <Copy> component onto the `copy` prop. It turns out this element isn't necessary for the `copy` prop. I removed it.